src/synthesizer/sed.py

```diff
@@ -1301,6 +1311,70 @@ def plot_spectra_as_rainbow(self, **kwargs):
         """
         return plot_spectra_as_rainbow(self, **kwargs)
 
+    # @accepts(velocity_fwhm=length/time, velocity_sigma=length/time)
```

Comment:
I'm going to create a Synthesizer specific version of this in light of the last few days 😅

Reply:
For now can you remove the commented out decorator?